src/debugger/mock-ui/debugger-cli.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+
+import { UISession } from "./UISession.js";
```

Comment:
This should better wrap Nuclide's `VsDebugSession` or use VSCode's utils instead.